Sources/Testing/Running/EntryPoint.swift

```diff
@@ -10,22 +10,120 @@
 
 private import TestingInternals
 
+#if canImport(Foundation) && !SWT_NO_ABI_ENTRY_POINT
+/// The type of the entry point to the testing library used by tools that want
+/// to remain version-agnostic regarding the testing library.
+///
+/// - Parameters:
+///   - argumentsJSON: A buffer to memory representing the JSON encoding of an
+///     instance of `__CommandLineArguments_v0`. If `nil`, a new instance is
+///     created from the command-line arguments to the current process.
+///   - eventHandler: An event handler to which is passed a buffer to memory
+///     representing each event and its context, as with ``Event/Handler``, but
+///     encoded as JSON.
+///
+/// - Returns: The result of invoking the testing library. The type of this
+///   value is subject to change.
+///
+/// This function examines the command-line arguments to the current process
+/// and then invokes available tests in the current process. The
+///
+/// - Warning: This function's signature and the structure of its JSON inputs
+///   and outputs have not been finalized yet.
+@_spi(ForToolsIntegrationOnly)
```

Comment:
The type alias needs to be outside the `!SWT_NO_ABI_ENTRY_POINT` check, right?

Reply:
This was intentional—if there's no ABI entry point, then there's no defined type for the ABI entry point either. :) Presumably a caller looking to use this symbol will configure their copy of swift-testing so that `SWT_NO_ABI_ENTRY_POINT` is not set. Does that make sense?

Reply:
Yes, sorry, I was confusing the conditional (again)...